src/main/java/step1/model/number/Number.java

```diff
@@ -0,0 +1,40 @@
+package step1.model.number;
+
+import step1.exception.BadRequestException;
+
+public class Number {
+	private static final String Numeric = "[0-9.]+";
```

Comment:
정적 상수는 대문자로 표기 되면 더 좋을 것 같습니다 😄 
```suggestion
	private static final String NUMERIC = "[0-9.]+";
```

Reply:
앗 다음 pr에서 리뷰 반영하겠습니다🙇‍♀️